docker-compose.prod.yml

```diff
@@ -1,5 +1,36 @@
 version: '3'
 services:
+    proxy:
+    image: jwilder/nginx-proxy
+    container_name: proxy
+    ports:
+      - "443:443"
+      - "80:80"
+    networks:
+      - backend
+    labels:
+      - "com.github.jrcs.letsencrypt_nginx_proxy_companion.nginx_proxy=true"
+    deploy:
+      restart_policy:
+        condition: on-failure
+    volumes:
+      - /var/run/docker.sock:/tmp/docker.sock:ro
+      - proxy/certs:/etc/nginx/certs
+      - proxy/vhost:/etc/nginx/vhost.d
+      - proxy/public:/usr/share/nginx/html
+      - ./proxy/nginx.tmpl:/app/nginx.tmpl
+
+  proxy-certs:
+    image: jrcs/letsencrypt-nginx-proxy-companion
+    networks:
+      - backend
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+      - proxy/certs:/etc/nginx/certs:rw
+      - proxy/vhost:/etc/nginx/vhost.d
+      - proxy/public:/usr/share/nginx/html
```

Comment:
Is this a named volume or a path?
https://docs.docker.com/compose/compose-file/#volumes